### PR TITLE
Aea 423 spine responses into operationoutcome

### PR DESCRIPTION
**Pull request by ri-ch**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: coordinator/src/routes/prescription/post-send-message.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ import * as translator from "../../services/translation/translation-service";
 import {Bundle} from "../../model/fhir-resources";
 import {sendData} from "../../services/spine-communication";
 import Hapi from "@hapi/hapi";
+import * as common from "../../services/translation/common"
```
Comment (dan-jones-iw) [inline coordinator/src/routes/prescription/post-send-message.ts:6]: seems sensible to import the `wrapInOperationOutcome` function the same way that `sendData` is imported on line 4

File: models/schemas/OperationOutcome-2XX.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+properties:
+  resourceType:
+    type: string
+    description: FHIR Resource Type.
+    default: "OperationOutcome"
+    readOnly: true
+  issue:
+    type: array
+    description: List of issues that have occurred.
+    minItems: 1
```
Comment (dan-jones-iw) [inline models/schemas/OperationOutcome-2XX.yaml:13]: is there always an issue?

File: specification/electronic-prescription-service-api.yaml [hunk 3]
```diff
@@ -203,10 +203,13 @@ paths:
         '200':
```
Comment (dan-jones-iw) [inline specification/electronic-prescription-service-api.yaml:203]: do 200 responses from other endpoints also return operation outcomes?

Comment (conner-jowitt) [reply specification/electronic-prescription-service-api.yaml:203]: No, /Convert returns xml, and /Prepare returns a Parameter resource.

File: models/schemas/OperationOutcome-2XX.yaml (hunk 2 above)
Comment (conner-jowitt) [reply models/schemas/OperationOutcome-2XX.yaml:13]: Yes, in the fhir spec, it is a required element.

**Pull request merged**